packages/process/src/node/process.ts

```diff
@@ -61,7 +63,17 @@ export abstract class Process {
     protected handleOnExit(event: IProcessExitEvent) {
         this._killed = true;
         const signalSuffix = event.signal ? `, signal: ${event.signal}` : '';
-        this.logger.info(`Process ${this.pid} has exited with code ${event.code}${signalSuffix}.`);
+        let cmd = this.command;
+        if (this.args) {
+            cmd += ' ' + this.args.join(' ');
+        }
+        const msg = `Process ${this.pid} has exited with code ${event.code}${signalSuffix} (${cmd}).`;
+
+        if (event.code !== 0) {
```

Comment:
When I kill the process (due to cancellation) the event.code is `null` and signal is `SIGKILL`. That shouldn't be logged with warning level, too.

Reply:
That is my opinion too.

Reply:
I think @akosyakov would agree here, too. I guess he was more concerned about erroneous exits. Maybe explicitly asking for `event.code > 0` helps?

Reply:
Well, you'll get the same problem when ripgrep doesn't find a match in the searched file.  Do you want to print a warning in that case?

Reply:
no

Reply:
then let's make it debug always